chromium_src/chrome/installer/mini_installer/BUILD.gni

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2023 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import("//brave/build/config.gni")
+import("//third_party/widevine/cdm/widevine.gni")
+
+template("modified_action") {
```

Comment:
I think it should be something like `template("brave_mini_installer_archive_action")`.

Reply:
Resolved in 6a007f1b6332c0285b72288fc9bf914e8fe7a4ab.